perses/tests/testsystems.py

```diff
@@ -1714,7 +1714,7 @@ def __init__(self, constraints=app.HBonds, premapped_json_dict=None, **kwargs):
         forcefield = app.ForceField(gaff_xml_filename, 'tip3p.xml', clinical_kinase_inhibitors_filename)
         d_smiles_to_oemol = {smiles : smiles_to_oemol(smiles, "MOL%d" % i)for i, smiles in enumerate(molecules)}
 
-        ## Generate and ffxml, then add to forcefield
+        # ## Generate and ffxml, then add to forcefield
```

Comment:
Did you mean to uncomment this code and pre-generate the parameters, or did you not want to pre-generate the parameters?

Reply:
That code is commented so whoever is looking at it can reproduce the error I was getting when loading the ffxml I already generated (https://github.com/choderalab/perses/blob/residue_templates/perses/data/clinical-kinase-inhibitors.xml). However, I wasn't sure if the problem was related to loading the template or generating the template. Therefore, to better understand the problem (and debug), I kept the code I used to generate the templates in there. Feel free to uncomment it if you think the problem occurs while generating the templates.